gef.py

```diff
@@ -4323,7 +4323,9 @@ def do_invoke(self, argv):
             extra = "dirty" if len(subprocess.check_output('git ls-files -m', cwd=gef_dir, shell=True).decode("utf8").strip()) else "clean"
```

Comment:
nit: Can you fix this line and previous to use double quotes? I believe in fixing nearby lines :)